react-validated-input-component/my-app/src/App.tsx

```diff
@@ -0,0 +1,3 @@
+import { ValidatedInput } from './ValidatedInput';
+
+<ValidatedInput />;
```

Comment:
I think you made an editing mistake. You lost your App component???

Reply:
Okay, it should be back now... I think i deleted on accident when removing the fragment